react/src/AuthGate.js

```diff
@@ -0,0 +1,209 @@
+"use strict";
+var __extends = (this && this.__extends) || (function () {
```

Comment:
Why is this file not ES6?

Reply:
Looks like some crap entered the PR, there supposed to be only TS file. I will clean this up.

Btw, you want ES6 along with ES5, I presume.

Reply:
Removed those 2 files, seems that they were produced by IDE.